README.md

```diff
@@ -787,6 +788,33 @@ The site configuration option `showRelatedInArticle` controls if this option is
 
 Likewise, the site configuration option `showRelatedInSidebar` controls if related content is shown on the sidebar. There is no corresponding option within a post to disable this.
 
+### Map
+
+## Creating and including a map
```

Comment:
Nitpick @imranakram: This and the "Options" heading should be fourth-level (4 `####`) not 2. If you look at the "rich diff" version you'll see they're at the wrong level.

Reply:
@rootwork Thanks, I've updated the heading level to 4 on `Creating and including a map` and `Options`